docs/server-side-search.rst

```diff
@@ -72,14 +72,14 @@ and then click on :guilabel:`Search Analytics`.
 API
 ---
 
-Search is exposed through our API that's proxied from the domain where your docs are being served.
-This is ``https://docs.readthedocs.io/_/api/v2/search`` for the ``docs`` project, for example.
+If you are using :doc:`/commercial/index` you will need to replace
+https://readthedocs.org/ with https://readthedocs.com/ in all the URLs used in the following examples.
```

Comment:
This should be under it's own section, probably `Read the Docs for Business Users` or similar

Reply:
This is describing the API and examples, not sure if I follow about having another section

Reply:
Like, we need a section of the docs explaining the 2 different sites and endpoints, and then probably link to it from here for more detail.